### DISCUSSION TITLE
Migration Guide from 8.0.0-beta.9 to 8.5.13

### DISCUSSION THREAD

**Discussion by yusufozturk** [TanStack/table#4406] [Q&A]
Hello,

Thank you for this library.

Is there any migration guide from 8.0.0-beta.9 to 8.5.13? Do we need to throw away all the customizations and start over with 8.5.13?

If there is also a migration guide from v7 to v8, would be really good. We have some legacy code as well.

Thanks.

Yusuf

**Comment by KevinVandy** [comment]
https://tanstack.com/table/v8/docs/guide/migrating
